app/api/domains/cho.py

```diff
@@ -787,19 +787,6 @@ async def login(
             ),
         }
 
-    if db_country == "xx":
-        # bugfix for old bancho.py versions when
-        # country wasn't stored on registration.
-        log(f"Fixing {login_data['username']}'s country.", Ansi.LGREEN)
-
-        await db_conn.execute(
-            "UPDATE users SET country = :country WHERE id = :user_id",
-            {
-                "country": geoloc["country"]["acronym"],
-                "user_id": user_info["id"],
-            },
-        )
-
```

Comment:
this is likely still in use on some older bancho.py installs, so would prefer to keep it. low risk/reward ratio

Reply:
I doubt 2 year old migrations are still relevant. I think we should define a baseline on how long migrations are maintained and comment that inside the code. I think more than 6 months is already a lot, users really should update bpy within 6 months. Also considering all those changes that happened recently I doubt someone is trying to migrate some 2 year old server